cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -1441,7 +1441,7 @@ protected ProgramStatus getProgramStatus(Id.Program id, ProgramType type) {
             return new ProgramStatus(id.getApplicationId(), id.getId(), HttpResponseStatus.NOT_FOUND.toString());
           } else {
             // program exists and not running. so return stopped.
-            return new ProgramStatus(id.getApplicationId(), id.getId(), ProgramController.State.STOPPED.toString());
+            return new ProgramStatus(id.getApplicationId(), id.getId(), "STOPPED");
```

Comment:
Can you open a JIRA about fixing/refactoring things around `ProgramStatus` and `ProgramRunState`. Seems pretty messy in how it works now (not due to this change).